package.json

```diff
@@ -10,7 +10,8 @@
   "main": "build/index.js",
   "types": "build/index.d.ts",
   "scripts": {
-    "build": "npm run check-types && cross-env NODE_ENV=production webpack --progress --mode production",
+    "build": "npm run check-types && npm run build:css && cross-env NODE_ENV=production webpack --progress --mode production",
+    "build:css": "postcss tailwind.css -o styles.css",
     "check-types": "tsc --sourceMap false --noEmit",
```

Comment:
are the tw styles built for both `example:application` and `storybook` tasks too? looks like only for `build` only so far

Reply:
u might want a separate task for production build css with `NODE_ENV=production` flag also, i think the current way is not purging the unused styles